blinky-statik/src/main/java/org/spideruci/analysis/statik/Statik.java

```diff
@@ -25,6 +28,21 @@
   public static final String RTJAR = "/rt.jar";
   public static final String JCEJAR = "/jce.jar";
   
+  private static void RUN_SOOT(String[] args) {
```

Comment:
These methods; RUN_SOOT, GET_ENTRY_METHODS, and GET_UNIT_GRAPH are all specific to Soot and have nothing to do with Blinky or its data structures. I am going to start setting them up as static method wrappers around Soot so that we don't have to deal with Soot's idiosyncrasies. Like it took me ages to figure out that `soot.Main.main(args);` actually ran Soot! Now, that is just horrible, and a clean method name around that lousy piece of code just makes things cleaner. If there is a better way to do this, i am all ears; like put them in their own class perhaps?